app/helpers/diff-to-document.ts

```diff
@@ -1,6 +1,6 @@
 import { helper } from '@ember/component/helper';
 
-const diffToDocument = helper(function diffToDocument([diff = '']: [string | undefined]) {
+export function parseDiff(diff: string = '') {
```

Comment:
@VasylMarchuk is there a downside to keeping this as it was? I think that's what we do in other helpers. Not a big deal but if we can stay consistent might as well.

Reply:
I want to reuse the parsing method that the helper uses in a different place (not the helper itself), maybe I can move it into a separate util?

Reply:
Ah, gotcha - yep let's move that to lib then! Makes it more obvious that it will be reused

Reply:
Done, moved into `utils/parse-diff-as-document` + added unit tests (copied from helper integration tests, but in unit syntax)